ui/raidboss/data/04-sb/trial/lakshmi.txt

```diff
@@ -0,0 +1,98 @@
+# Emanation (normal mode)
+# Lakshmi
+
+hideall "--Reset--"
+hideall "--sync--"
+hideall "Alluring Arm"
+
+# -ii 2157 248B 248D
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start" sync /Engage!/ window 0,1
+3.9 "Vril" sync /:Dreaming Kshatriya:2482:/ window 3.9,5
+13.0 "Stotram" sync /:Lakshmi:2483:/ window 13,5
+16.1 "Inner Demons" # sync /:Dreaming Kshatriya:258D:/
+19.0 "Aether Drain" sync /:Vril:248D:/
+29.3 "Tail Slap" # sync /:Dreaming Kshatriya:258C:/
+
+# We can't sync to abilities here to make the loop visible,
+# since two adds are present, and the timeline would jerk around.
+60.0 "--sync--" sync /.*/ window 0,30 jump 30
```

Comment:
Hmm.  This is a lot of timeline resyncing just to keep a loop going.  Is there a single ability that can loop back here, or would it make sense to just move `100.0` farther into the future to be `1000.0`?

One future feature that I think would be nice would be to have a pause/break sort of keyword where you could insert like `60.0 pause` or something and the timeline would pause there until it reached another sync and no abilities past that time would be shown past there.  We could use that for phase breaks everywhere.

Reply:
No, there's nothing usable. (The `/.*/` construction was borrowed from EX, which is admittedly *fairly* old.) The adds may or may not be in sync, depending on which tank they target and how far they have to travel to start auto-attacking. In this case it's likely best to just start the syncs from 1000.